sensu/files/plugins/check-ip-connectivity.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -e
+
+STATE_OK=0
+STATE_WARNING=1
+STATE_CRITICAL=2
+STATE_UNKNOWN=3
+STATE_DEPENDENT=4
+
+#target=
+
+count=$(ping -c 1 10.13.37.83 | grep icmp* | wc -l)
```

Comment:
10.13.37.83 - Hostname could be parameterized but as we have single host (this could be postponed).